numpy/core/tests/test_multiarray.py

```diff
@@ -1798,7 +1787,8 @@ def test_sort(self):
             c = b.copy()
             c.sort(kind=kind)
             assert_equal(c, a, msg)
-
+            
+    def test_sort_structured(self):
         # test record array sorts.
         dt = np.dtype([('f', float), ('i', int)])
         a = np.array([(i, i) for i in range(101)], dtype=dt)
```

Comment:
I can't stick a comment on it, but the `msg` on line1807 shouldn't say `object`. Again, it can be shortened, since the test name contains the word `structured` already